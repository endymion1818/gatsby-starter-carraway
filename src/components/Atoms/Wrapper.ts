import styled from 'styled-components'

export interface IWrapperProps {
  backgroundColour?: string
  textColour?: string
  paddingTop?: string
  paddingBottom?: string
}

const Wrapper = styled.div<IWrapperProps>`
  width: 100%;
  background-color: ${({ backgroundColour }) => backgroundColour};
  color: ${({ textColour }) => textColour};
  padding-top: ${({ paddingTop }) => paddingTop};
  padding-bottom: ${({ paddingBottom }) => paddingBottom};
`

export default Wrapper
