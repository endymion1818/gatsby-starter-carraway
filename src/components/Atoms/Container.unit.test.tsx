import { cleanup, render } from '@testing-library/react'
import React from 'react'
import Wrapper from './Wrapper'

describe('<Wrapper />', () => {
  beforeEach(cleanup)
  it('Should render', () => {
    const { container } = render(<Wrapper />)
    expect(container).toMatchSnapshot()
  })
})
