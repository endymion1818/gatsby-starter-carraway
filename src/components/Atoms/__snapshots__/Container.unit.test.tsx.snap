// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> Should render 1`] = `
.c0 {
  max-width: 36em;
  margin-left: auto;
  margin-right: auto;
  padding-left: 1rem;
  padding-right: 1rem;
}

<div>
  <div
    class="c0"
  />
</div>
`;
